Dockerfile

```diff
@@ -7,6 +7,8 @@ RUN apt-get -q -y update && \
     ca-certificates \
     ssh \
     git \
+    parallel \
+    && echo "will cite" | parallel --citation \
```

Comment:
Is this something customer also need to run?

Reply:
Nope, not any more with my latest changes.